custom/templates/DefaultRevamp/403.tpl

```diff
@@ -1,20 +1,23 @@
-{include file='header.tpl'}
-    <div class="ui container" id="error-403">
-      <div class="ui segment">
-        <h2 class="ui header">{$403_TITLE}</h2>
-        <div class="ui divider"></div>
-        <p>{$CONTENT}</p>
-        {if !isset($LOGGED_IN_USER)}<p>{$CONTENT_LOGIN}</p>{/if}
-        <div class="ui buttons">
-          <button class="ui primary button" onclick="javascript:history.go(-1)">{$BACK}</button>
-          <div class="or"></div>
-          {if isset($LOGGED_IN_USER)}
-            <a class="ui positive button" href="{$SITE_HOME}">{$HOME}</a>
-          {else}
-            <a class="ui positive button" href="{$LOGIN_LINK}">{$LOGIN}</a>
-          {/if}
-        </div>
-      </div>
-    </div>
-  </body>
-</html>
\ No newline at end of file
+{extends 'layouts/basic.tpl'}
+
+{block 'body'}
+  <div class="ui container">

```

Comment:
Removing the outer container element gets essentially this result. I opted to keep it in all pages including the error pages for the sake of consistency and so if a user added a graphic to the page or something it would keep the content width, could probs remove though :)
![image](https://user-images.githubusercontent.com/11805258/163702594-f46f72c0-a1ff-49ac-b0fd-9e116cbe74fb.png)